Source/WebKit/UIProcess/Cocoa/VideoFullscreenManagerProxy.mm

```diff
@@ -894,6 +900,23 @@ - (BOOL)prefersStatusBarHidden
     m_page->send(Messages::VideoFullscreenManager::ReturnVideoContentLayer(contextId));
 }
 
+void VideoFullscreenManagerProxy::returnToStandby(PlaybackSessionContextIdentifier contextId)
```

Comment:
is "standby" a well-known term? it doesn't mean anything to me, and I wonder if a two or three word phrase might work better

Reply:
I _wish_ I could name it something like "returnVideoContentLayer()", except that's already taken. :-/

Reply:
And "standby" is a term of art for "PrepareForAutoPiPFromInline".

Reply:
I'll rename this "returnVideoView()" so that it does what it says on the box.